Add new fields to Location model

mforner13 opened the pull request:
This PR is the first of two which will replace the fields in the Location and Masterclass tables due to the following design changes:
- A masterclass will take place either remotely or in person
- Physical locations will be fetched from the google places API, rather than input manually

The three remote fields have been added to the masterclass model, rather than location because these will be individual to each masterclass, whereas multiple masterclasses can have the same location.

I have included the boolean `is_remote` to the masterclass model because remote and in person masterclasses will be displayed differently in the app.

**TODO**
- Update templates to display Location data from new fields
- Create another migration to remove the old Location fields

jonodrew commented on app/models.py:
```diff
@@ -45,6 +45,12 @@ class Location(db.Model):
     postcode = db.Column(db.String(8), index=True)  # A postcode in the UK can't have more than 8 characters inc. space
+    is_remote = db.Column(db.Boolean, index=True)
+    remote_url = db.Column(db.String, index=True)
+    remote_joining_instructions = db.Column(db.String, index=True)
```
Not sure this belongs here

jonodrew replied:
If we want locations to be re-useable maybe move this to the actual Masterclass? As it'll be unique data associated with each Masterclass

mforner13 committed:
Update location and masterclass fields

Update location fields to match data received from the google places API and add fields related to remote masterclasses to the masterclass table.

mforner13 commented:
@jonodrew I have updated the model changes as we discussed last week

jonodrew reviewed (approved):
Looks good to me :+1:
